spec/keymap-manager-spec.coffee

```diff
@@ -335,6 +339,74 @@ describe "KeymapManager", ->
 
           expect(events).toEqual ['input:d', 'input:o', 'dog']
 
+    describe "when the a binding specifies a keyup handler", ->
+      [events, elementA] = []
+
+      beforeEach ->
+        elementA = appendContent $$ ->
+          @div class: 'a'
+
+        events = []
+        elementA.addEventListener 'y-command', (e) -> events.push('y-keydown')
+        elementA.addEventListener 'y-command-ctrl-up', (e) -> events.push('y-ctrl-keyup')
+        elementA.addEventListener 'x-command-ctrl-up', (e) -> events.push('x-ctrl-keyup')
+        elementA.addEventListener 'y-command-y-up-ctrl-up', (e) -> events.push('y-up-ctrl-keyup')
+        elementA.addEventListener 'abc-secret-code-command', (e) -> events.push('abc-secret-code')
+
+        keymapManager.add "test",
+          ".a":
+            "ctrl-y": "y-command"
+            "ctrl-y ^ctrl": "y-command-ctrl-up"
+            "ctrl-x ^ctrl": "x-command-ctrl-up"
+            "ctrl-y ^y ^ctrl": "y-command-y-up-ctrl-up"
+            "a b c ^b ^a ^c": "abc-secret-code-command"
+
+      it "dispatches the command when a matching keystroke precedes it", ->
+        keymapManager.handleKeyboardEvent(buildKeydownEvent('y', ctrl: true, target: elementA))
+        keymapManager.handleKeyboardEvent(buildKeyupEvent('y', ctrl: true, cmd: true, shift: true, alt: true, target: elementA))
+        keymapManager.handleKeyboardEvent(buildKeyupEvent('ctrl', target: elementA))
+        advanceClock(keymapManager.getPartialMatchTimeout())
+        expect(events).toEqual ['y-up-ctrl-keyup']
+
+      it "dispatches a command when modifier is lifted before the character", ->
```

Comment:
can you make the language consistent here... "the command" "a command" "keyup command" (I think "the command" would be the best)